code/modules/surgery/tools.dm

```diff
@@ -111,18 +111,9 @@
 	if(contents.len)
 		to_chat(user, "<span class='notice'>[src] already has something inside it.</span>")
 		return
-	if(isorgan(I))
-		var/obj/item/organ/O = I
-		if(O.status != ORGAN_ORGANIC)
-			to_chat(user, "<span class='notice'>[src] can only hold organic body parts!</span>")
-			return
-	else if(isbodypart(I))
-		var/obj/item/bodypart/BP = I
-		if(BP.status != BODYPART_ORGANIC)
-			to_chat(user, "<span class='notice'>[src] can only hold organic body parts!</span>")
-			return
-	else
-		return
+	if(!isorgan(I) && !isbodypart(I))
+		to_chat(user, "<span class='notice'>[src] can only hold body parts!</span>")
+    	return

```

Comment:
Do not use spaces for indenting. Only tabs.
One tab for at the beginning of line 114.
Two tabs at the beginning 115 and 116.
Remove all spaces in front of the actual code. (Do not alter the code itself, just the stuff in front of it)
**(tab)**`if(!isorgan(I) && !isbodypart(I))`
 **(tab)(tab)**`to_chat(user, "<span class='notice'>[src] can only hold body parts!</span>")`
 **(tab)(tab)**`return`